prime-angular-frontend/src/app/core/services/console-logger.service.ts

```diff
@@ -24,7 +26,7 @@ export class ConsoleLoggerService extends AbstractLoggerService {
    * Prints the logging information, but ONLY if not in production.
    */
   protected send(type: string, params: { msg?: string, data?: any[] }) {
-    if (!environment.production || type === 'error' || type === 'warn') {
+    if (this.config.environmentName !== 'prod' || type === 'error' || type === 'warn') {
```

Comment:
Bonus: using config allows us to know the actual environment versus whether the web client was build using `--prod` so we will now have stack traces in dev and test.